src/components/structures/MatrixChat.tsx

```diff
@@ -394,7 +395,18 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         }
 
         const crossSigningIsSetUp = cli.getStoredCrossSigningForUser(cli.getUserId());
-        if (crossSigningIsSetUp) {
+
+        // Pulls "default" value for e2e from well-known config.
+        // This can be used to check if encryption is enabled/disabled in home-server.
+        const encryptionIsEnabled = () => {
+            if (!SettingsStore.getValue(UIFeature.LookForWellKnownConfigFromHomeServer)) {
+                return null;
+            }
+            console.log("well-known config has default e2e value set to::: ", getE2EEWellKnown().default);
+            return (getE2EEWellKnown().default === true) ? true : false;
```

Comment:
Please remove the redundant ? true : false

Reply:
This has been actioned in new commit.